generator-jhipster-gomicro/generators/server/generator.mjs

```diff
@@ -0,0 +1,304 @@
+import chalk from "chalk";
+import yosay from 'yosay';
+import ServerGenerator from "generator-jhipster/generators/server";
+import { askForServerSideOpts } from './prompts.mjs';
+
+export default class extends ServerGenerator {
+  constructor(args, opts, features) {
+    super(args, opts, features);
+
+    if (this.options.help) return;
+
+    if (!this.options.jhipsterContext) {
+      throw new Error(
+        `This is a JHipster blueprint and should be used only like ${chalk.yellow(
+          "jhipster --blueprints test"
+        )}`
+      );
+    }
+  }
+
+  get [ServerGenerator.INITIALIZING]() {
+    return {
+      // ...super.initializing,
+      // async initializingTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.PROMPTING]() {
+    return {
+      // ...super.prompting,
+      // async promptingTemplateTask() {},
+      prompting() {
+        // Have Yeoman greet the user.
+        this.log(
+          yosay(
+            `${chalk.red('golang-blueprint')}`
+          )
+        );
+      },
+      askForServerSideOpts
+    };
+  }
+
+  get [ServerGenerator.CONFIGURING]() {
+    return {
+      // ...super.configuring,
+      // async configuringTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.COMPOSING]() {
+    return {
+      // ...super.composing,
+      // async composingTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.LOADING]() {
+    return {
+      // ...super.loading,
+      // async loadingTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.PREPARING]() {
+    return {
+      // ...super.preparing,
+      // async preparingTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.CONFIGURING_EACH_ENTITY]() {
+    return {
+      // ...super.configuringEachEntity,
+      // async configuringEachEntityTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.LOADING_ENTITIES]() {
+    return {
+      // ...super.loadingEntities,
+      // async loadingEntitiesTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.PREPARING_EACH_ENTITY]() {
+    return {
+      // ...super.preparingEachEntity,
+      // async preparingEachEntityTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.PREPARING_EACH_ENTITY_FIELD]() {
+    return {
+      // ...super.preparingEachEntityField,
+      // async preparingEachEntityFieldTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.PREPARING_EACH_ENTITY_RELATIONSHIP]() {
+    return {
+      // ...super.preparingEachEntityRelationship,
+      // async preparingEachEntityRelationshipTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.POST_PREPARING_EACH_ENTITY]() {
+    return {
+      // ...super.postPreparingEachEntity,
+      // async postPreparingEachEntityTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.DEFAULT]() {
+    return {
+      // ...super.default,
+      // async defaultTemplateTask() {},
+    };
+  }
+
+  get [ServerGenerator.WRITING]() {
+    return {
+      // ...super.writing,
```

Comment:
remove this comment  under "get [ServerGenerator.WRITING]()" as it is not required in future as well.